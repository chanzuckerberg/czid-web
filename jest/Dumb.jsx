import React from "react";

const Dumb = ({ text1, text2 }) => (
  <div>
    {text1 ? <div>Text 1 is: {text1}</div> : null}
    {text2 ? <div>Text 2 is: {text2}</div> : null}
  </div>
);

export default Dumb;
