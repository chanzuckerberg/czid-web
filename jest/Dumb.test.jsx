import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

import Dumb from "./Dumb";

describe("Dumb", () => {
  describe("when `text1` and `text2` are present", () => {
    it("renders correctly", () => {
      const props = {
        text1: "test-1",
        text2: "test-2",
      };

      const wrapper = shallow(<Dumb {...props} />);

      expect(wrapper.getElement()).toMatchSnapshot();
    });
  });
});
