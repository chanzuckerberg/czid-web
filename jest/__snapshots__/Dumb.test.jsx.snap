// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dumb when \`text1\` and \`text2\` are present renders correctly 1`] = `
<div>
  <div>
    Text 1 is: 
    test-1
  </div>
  <div>
    Text 2 is: 
    test-2
  </div>
</div>
`;
